risc0/cargo-risczero/Cargo.toml

```diff
@@ -33,7 +33,8 @@ reqwest-middleware = "0.2"
 reqwest-retry = "0.3"
 risc0-build = { workspace = true }
 risc0-r0vm = { workspace = true, optional = true }
-risc0-zkvm = { workspace = true, optional = true }
+risc0-zkvm = { path = "../zkvm", default-features = false, features = ["std", "getrandom"] }
+risc0-zkvm-methods = { path = "../zkvm/methods", default-features = false }
```

Comment:
Unfortunately this isn't publishable, we won't be able to publish this on crates.io.  We need to come up with some other clever way to embed the guest.

Reply:
Oh... Thanks for noting this! (and the quick review 😄)